osu.Game.Tests/Visual/Ranking/TestSceneStatisticsPanel.cs

```diff
@@ -44,5 +60,24 @@ public void TestNullScore()
                 Score = { Value = score }
             };
         });
+
+        private static List<HitEvent> createPositionDistributedHitEvents()
+        {
+            var hitEvents = new List<HitEvent>();
+            // Use constant seed for reproducibility
+            var random = new Random(0);
+
+            for (int i = 0; i < 500; i++)
+            {
+                double angle = random.NextDouble() * 2 * Math.PI;
+                double radius = random.NextDouble() * 0.5f * OsuHitObject.OBJECT_RADIUS;
```

Comment:
for the record, this is not how you properly do uniform sampling over a unit sphere in polar coordinates. to get actual uniform sampling, the square root of the random factor in the radius has to be taken, or otherwise the samples are biased towards points closer to the centre.

that said this doesn't matter at all in this context so i've left it in.

Reply:
Yeah I noticed that but I thought it makes sense to have it biased towards the center as real hits will also be

Reply:
that is also part of the reason why i left it as is.